static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -134,6 +139,9 @@ const createAdsConfig = (
         adTagParameters: {
             iu: config.get('page.adUnit'),
             cust_params: encodeURIComponent(constructQuery(custParams)),
+            cmpGdpr: tcfData.gdprApplies ? 1 : 0,
```

Comment:
Should this not simply check the existence of `state.tcfv2`?
Does `gdprApplies` come from the default IAB API?

Reply:
yap [comes from IAB](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md#what-does-the-gdprapplies-value-mean) and added to latest cmp release